### PR TITLE
Add measures for Activity Stream, Search, and Contextual Services

### PR DESCRIPTION
**Pull request by fbertsch**
This covers the tables we outlined in bug 1711955. We will probably follow this up with additional tables for both CS and AS.

### CODE REVIEW

File: search/views/search_clients_engine_sources_daily.view.lkml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+include: "//looker-hub/search/views/search_clients_engines_sources_daily.view.lkml"
+
+view: +search_clients_engines_sources_daily {
```
Comment (fbertsch) [inline search/views/search_clients_engine_sources_daily.view.lkml:3]: These measures I added deserve a close eye from the search team:
- Do these names make sense for what the query is doing?
- Are the descriptions correct?
- Do we need to remove some or add more variants?

Note closely what the UI gives you in the info field, which does include the SQL fields being summed (those will not, however, be available to see in visualizations):
Screen Shot 2021-05-26 at 11 21 09 PM

Comment (mmccorks) [reply search/views/search_clients_engine_sources_daily.view.lkml:3]: This looks good to me.

**Review by mmccorks — APPROVED** [review]
LGTM

**Comment by fbertsch** [comment]
Thank you @mmccorks!

**Pull request merged**